### ISSUE TITLE
 DataGrid usage and error messages with DataAnnotations

### ISSUE THREAD

**Issue by Weidmann16** [Megabit/Blazorise#4437]
Hello again

I am currently using DataAnnotations for the models
![image](https://user-images.githubusercontent.com/74007151/210806977-71636f32-e53c-4175-972f-db94ee89f7fd.png)

Could the datagrid show the DataAnnotations error message?
![image](https://user-images.githubusercontent.com/74007151/210807206-a11b990e-9e71-40e1-b0cc-660326b129c8.png)
![image](https://user-images.githubusercontent.com/74007151/210807778-4ba899e7-5bf2-48da-8f43-3a03dd1aface.png)

instead of using the one that is placed manually as the documentation shows
![image](https://user-images.githubusercontent.com/74007151/210807323-3fa9f6bb-5025-4570-a240-e4e7d75ee0c1.png)

in a nutshell, I want the DataAnnotations message to be displayed

I will be careful

**Comment by stsrki** [comment]
Just remove the Validator. It has higher priority if it's defined.

**Comment by David-Moreira** [comment]
Closing issue. 
Feel free to reopen the issue if you feel that your case has not been resolved successfully.
